boefjes/plugins/kat_censys/normalize.py

```diff
@@ -9,50 +9,47 @@
     IPPort,
     Protocol,
     PortState,
-    IPAddressV4,
-    IPAddressV6,
     Network,
 )
+from octopoes.models.ooi.service import IPService, Service
 from octopoes.models.ooi.software import Software, SoftwareInstance
-from octopoes.models.ooi.web import HTTPHeader
+from octopoes.models.ooi.web import (
+    HTTPHeader,
+    HTTPResource,
+    Website,
+    IPAddressHTTPURL,
+)
+
 from boefjes.job_models import NormalizerMeta
 
 
 def run(normalizer_meta: NormalizerMeta, raw: Union[bytes, str]) -> Iterator[OOI]:
     results = json.loads(raw)
-    ooi = Reference.from_str(normalizer_meta.boefje_meta.input_ooi)
+    ip_ooi_reference = Reference.from_str(normalizer_meta.boefje_meta.input_ooi)
 
-    network = Network(name="internet").reference
+    network_reference = Network(name="internet").reference
```

Comment:
Maybe fetch the network from the ip_ooi_reference?

Reply:
Good catch!